tests/queries/0_stateless/02161_addressToLineWithInlines.sh

```diff
@@ -0,0 +1,109 @@
+#! /bin/env bash
+# Tags: no-tsan, no-asan, no-ubsan, no-msan, no-debug
+# tags are copied from 00974_query_profiler.sql
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+export CLICKHOUSE_DATABASE=system
+# shellcheck source=../shell_config.sh
+. "$CURDIR"/../shell_config.sh
+
+MAX_FAILED_COUNT=10
+MAX_RETRY_COUNT=10
+log_comment="02161_testcase_$(date +'%s')"
+
+check_exist_sql="SELECT count(), query_id FROM system.trace_log WHERE trace_type IN ('CPU', 'Real') AND query_id IN (
+    SELECT query_id FROM query_log WHERE log_comment = '${log_comment}' ORDER BY event_time DESC LIMIT 1
+) GROUP BY query_id"
+
+declare exist_string_result
+declare -A exist_result=([count]=0 [query_id]="")
```

Comment:
Why not just use regular variables. It will be easier to read this script.

Reply:
Some history... I have changed this test many times, the CI have not successed any times... I just plan to refactor it after it finally success...